.github/workflows/push.yml

```diff
@@ -18,7 +18,6 @@ jobs:
        python -m pip install --upgrade pip
        pip install -r requirements.txt
 
-
    compile:
     runs-on: windows-latest
```

Comment:
@omanimo  you want to use a matrix here, I run macos, marvin runs fedora and the server runs alpine ensure it compiles on all of those please

Reply:
Noted @zacck. Let me get on it.